.gitignore

```
@@ -1,3 +1,4 @@
 .DS_Store
 .idea
 *.iml
+node_modules
```

(Missing end of file newline)

```suggestion
node_modules

```